src/main/java/com/theateam/waaprojectspringbackend/controller/CustomersController.java

```diff
@@ -53,15 +60,45 @@ public void updateOffer(@PathVariable Long offerId, @RequestBody UpdateOfferDto
     }
 
     @GetMapping("/saved-properties")
-    public String getSavedProperties() {
-        return "Greetings from Customers GET saved-properties!";
+    public ResponseEntity<List<String>> getSavedProperties() {
+        Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
+        String userEmail = authentication.getName();
+        Optional<User> user = userService.getUserByEmail(userEmail);
```

Comment:
here u can just leave `userService.getUserByEmail(userEmail).orElseThrow` then u dont need to check for user if found